core/DataAccess/RawLogDao.php

```diff
@@ -45,38 +60,152 @@ public function updateConversions(array $values, $idVisit)
     /**
      * @param string $from
      * @param string $to
-     * @param array $fields
-     * @param int $fromId
-     * @param int $limit
-     * @return array[]
+     * @return int
      */
-    public function getVisitsWithDatesLimit($from, $to, $fields = array(), $fromId = 0, $limit = 1000)
+    public function countVisitsWithDatesLimit($from, $to)
     {
-        $sql = "SELECT " . implode(', ', $fields)
+        $sql = "SELECT COUNT(*) AS num_rows"
              . " FROM " . Common::prefixTable('log_visit')
-             . " WHERE visit_first_action_time >= ? AND visit_last_action_time < ?"
-             . " AND idvisit > ?"
-             . sprintf(" LIMIT %d", $limit);
+             . " WHERE visit_first_action_time >= ? AND visit_last_action_time < ?";
```

Comment:
here I'd be tempted to change `visit_first_action_time` to `visit_last_action_time` so this query uses the index and is faster.